{"name": "GET Non-Existent User", "status": "failed", "statusDetails": {"message": "AssertionError: assert 401 == 404\n +  where 401 = <Response [401]>.status_code", "trace": "@allure.title(\"GET Non-Existent User\")\n    @allure.severity(allure.severity_level.MINOR)\n    def test_get_user_not_found():\n        url = f\"{BASE_URL}/users/9999\"\n        response = requests.get(url, headers=get_headers())\n    \n>       assert response.status_code == 404\nE       assert 401 == 404\nE        +  where 401 = <Response [401]>.status_code\n\ntests/test_users_api.py:50: AssertionError"}, "start": 1751508632991, "stop": 1751508633377, "uuid": "7ab3581b-9487-49d8-a3fc-829f1deca90e", "historyId": "314fdc1812f35ca4a065312d1edd7e33", "testCaseId": "314fdc1812f35ca4a065312d1edd7e33", "fullName": "tests.test_users_api#test_get_user_not_found", "labels": [{"name": "severity", "value": "minor"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_users_api"}, {"name": "host", "value": "codespaces-2886b5"}, {"name": "thread", "value": "5609-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_users_api"}]}